doc/pgtap.mmd

```diff
@@ -4524,8 +4524,8 @@ pass the type as either "varchar(64)" or "character varying(64)". Example:
 
     SELECT col_type_is( 'myschema', 'sometable', 'somecolumn', 'timespantz(3)' );
```

Comment:
Not part of the patch, but would you please fix the typo in `timespantz`.

But `timestamp(3)` doesn't even match the paragraph above which talks about `varchar(64)`. Maybe it's used here because `timestamptz`/`timestamp with time zone` is similar to `varchar`/`character varying`.

Reply:
Done in b0184e4.